### PR TITLE
Build: don't assume the browser environment; drop old JSHint options

### PR DESCRIPTION
**Pull request by mzgol**
Fixes gh-345

This turned out to be easy. No source changes required. :)

### CODE REVIEW

File: src/.jshintrc [hunk 1]
```diff
@@ -6,21 +6,21 @@
 	"expr": true,
 	"immed": true,
 	"noarg": true,
-	"onevar": true,
 	"quotmark": "double",
-	"smarttabs": true,
-	"trailing": true,
```
Comment (mzgol) [inline src/.jshintrc]: These options are no longer supported by JSHint. JSCS should be used to check such things.

**Comment by mzgol** [comment]
cc @gibson042. This PR is very simple, fortunately. :)

Could we land it & then release a new Sizzle? That would open the doors to update Sizzle in jQuery & move Edge to the main browser set in TestSwarm.

**Comment by gibson042** [comment]
LGTM :+1:

**Comment by mzgol** [comment]
And... I made a typo in "Closes" (I wrote "Closse"). Landed at d5905fd.

**Pull request closed without merging**